{{cookiecutter.project_name}}/noxfile.py

```diff
@@ -254,22 +261,30 @@ def xdoctest(session: Session) -> None:
     session.run("python", "-m", "xdoctest", package, *args)
 
 
-@nox.session(python="3.8")
-def docs(session: Session) -> None:
+def clean_build_path() -> None:
+    """Clean build path."""
+    build_dir = Path("docs", "_build")
+    if build_dir.exists():
+        shutil.rmtree(build_dir)
+
+
```

Comment:
I appreciate the attempt to DRY up the sessions. But let's keep the three lines duplicated in each session, instead of defining and calling a helper function. This change actually adds more lines than it removes. It also adds a top-level module definition that is not a session. Splitting the session definitions into two functions is not worth it for a handful of lines, especially when everything else still happens inline.

Reply:
Removed the helper function.